src/resources/views/crud/form_content.blade.php

```diff
@@ -5,7 +5,7 @@
     @include('crud::inc.show_tabbed_fields')
     <input type="hidden" name="_current_tab" value="{{ Str::slug($crud->getTabs()[0]) }}" />
 @else
-  <div class="card">
+  <div class="card border-0">
     <div class="card-body row">
       @include('crud::inc.show_fields', ['fields' => $crud->fields()])
     </div>
```

Comment:
Same here - issue doesn't belong in this view, but it's on both CREATE and UPDATE operations. I think it might be related to our `CrudField JS Library` in fact.

When I click a field (any field), it triggers a JS error in the console: `Uncaught TypeError: No method named "destroy"`. Those quickly add up:
![CleanShot 2023-02-26 at 11 02 33](https://user-images.githubusercontent.com/1032474/221401465-e0d87d59-271a-46ad-8cd7-805a641d4993.png)

Reply:
Will close when https://github.com/Laravel-Backpack/CRUD/pull/4860/files#r1112727805 is fixed

Reply:
Fixed on `v6`. I just pulled the fix to the `coreuiv4` branch too.